edx_exams/apps/api/v1/views.py

```diff
@@ -453,3 +511,19 @@ def put(self, request, attempt_id):
             status=status.HTTP_400_BAD_REQUEST,
             data={'detail': f'Unrecognized action "{action}"'}
         )
+
+    def post(self, request):
```

Comment:
It looks like something went wrong during your rebase - this is a function that I added, so it shouldn't be showing up here.

Reply:
Yeah I'm pretty sure I did the rebase incorrectly. Should I remove this function?

Reply:
Nope don't remove it! You need to pull the latest changes from main and then rebase your branch off of main.